tests/Update/RecipePatcherTest.php

```diff
@@ -38,13 +38,31 @@ public function testGeneratePatch(array $originalFiles, array $newFiles, string
         // original files need to be present to avoid patcher thinking they were deleting and skipping patch
         foreach ($originalFiles as $file => $contents) {
             touch(FLEX_TEST_DIR.'/'.$file);
+            if ($file === '.gitignore') {
+                file_put_contents(FLEX_TEST_DIR.'/'.$file, $contents);
+            }
+        }
+
+        // make sure the test directory is a git repo
+        (new Process(['git', 'init'], FLEX_TEST_DIR))->mustRun();
+        (new Process(['git', 'config', 'commit.gpgsign', 'false'], FLEX_TEST_DIR))->mustRun();
```

Comment:
Is this strictly needed?

Reply:
Yeah I'm not sure about this, creating the git repo failed in the ci environnement on my first try so I copied this from [RecipePatcher](https://github.com/symfony/flex/blob/d84cd84094be56d55cfd5db080db4c40a6dcce16/src/Update/RecipePatcher.php#L113) to mimic the way the git repo is created. But this command about gpgsign might not be needed. I'll try and remove it and see if the ci tests still pass :)

Reply:
Seems like it passed !